CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -285,11 +292,11 @@ private void DestroyUniformBuffers() {
         }
     }
     
-    private void DestroyDescriptorSets(){
-        // TODO HYDRA
-        //pDescriptorSets empty list
-                
-        //vkDestroyDescriptorSetLayout(cvkDevice.GetDevice(), hDescriptorLayout, null);
+    private void DestroyDescriptorSets(CVKSwapChain cvkSwapChain){
+        if (pDescriptorSets != null) {
+            vkFreeDescriptorSets(cvkDevice.GetDevice(), cvkSwapChain.GetDescriptorPoolHandle(), pDescriptorSets);
+            pDescriptorSets = null;
+        }
```

Comment:
Have you tested this code?  When I had code like that in FPS I got validation errors when it ran.

Reply:
no errors it works